pkg/clients/sns/topic_test.go

```diff
@@ -33,10 +33,13 @@ import (
 
 var (
 	empty             = ""
+	falseFlag         = false
```

Comment:
ok I should say this and the next lines are kind of weird, but who am I to judge weirdness :)

Reply:
Agree, i just tried to stick with the pattern :see_no_evil: 
I'm not so experienced in golang yet and didn't want to rework this in the same pr.

Reply:
can we proceed then with it please? :innocent: =

Reply:
Can we merge then please :) ? @danilaplee @holooloo

Reply:
Yes, sure
Do it

Reply:
ok